bedrock/base/management/commands/update_www_config.py

```diff
@@ -49,10 +50,22 @@ def output(self, msg):
             print(msg)
 
     def handle(self, *args, **options):
+        # Grab the existing monitor waitlist value so if something fails with the API, we don't lose it.
+        try:
```

Comment:
could move this into the `set_monitor_waitlist` method since the DB refresh happens later.